src/backend/distributed/executor/multi_server_executor.c

```diff
@@ -212,7 +212,7 @@ InitTaskExecution(Task *task, TaskExecStatus initialTaskExecStatus)
 	taskExecution->jobId = task->jobId;
 	taskExecution->taskId = task->taskId;
 	taskExecution->nodeCount = nodeCount;
-	taskExecution->connectPollCount = 0;
+	taskExecution->connectStartTime = 0;
```

Comment:
It's a correct value for either, so ...?  We're not super strict in the codebase about adding casts for every promotion, and I don't really see the benefit to start with something as innocuous as above.